File: ProjectPendragon/src/app/models/character/wealth.ts
export class Wealth {
    id: string;
    libra: number;
    shilling: number;
    denari: number;
    
    constructor() {
        this.id = crypto.randomUUID();
        this.libra = 0;
        this.shilling = 0;
        this.denari = 0;
    }
}

File: ProjectPendragonBackend/Models/Wealth.cs
namespace ProjectPendragonBackend.Models
{
    public class Wealth
    {
        public Guid Id { get; set; }
        public Guid CharacterId { get; set; }
        public int Libra { get; set; }
        public int Shilling { get; set; }
        public int Denari { get; set; }
    }
}